alacritty_terminal/src/tty/unix.rs

```diff
@@ -239,6 +240,11 @@ pub fn from_fd(config: &Options, window_id: u64, master: OwnedFd, slave: OwnedFd
                 return Err(Error::new(ErrorKind::Other, "Failed to set session id"));
             }
 
+            // Don't fail
```

Comment:
This comment should be reworked too and is missing punctuation. I recommend looking at the suggestion of my other comment as reference.